### DISCUSSION TITLE
LAN vs. WAN data

### DISCUSSION THREAD

**Discussion by tonygauderman** [bsmithio/OPNsense-Dashboard#20] [General]
With the latest json files, it seems like the LAN interface is showing the traffic in the wrong direction (transmit and receive switched).

I changed the JSON to to both show the throughput in the right direction and show the direction in the same order as the WAN (may not be the intent???)

        {
          "id": "organize",
          "options": {
            "excludeByName": {},
            "indexByName": {
              "Time": 0,
              "bytes_recv": 1,
              "bytes_sent": 2
            },
            "renameByName": {
              "bytes_recv": "Bytes Recv - This Month",
              "bytes_sent": "Bytes Sent - This Month"
            }
          }
        }

I did a similar change for the Traffic Graph.  All of those changes  may not make sense to everyone, but to me, if the direction is the same color in each interface, which makes sense, it's easier to read with the throughputs being displayed in the same order as well.  This does resolve the traffic showing in the wrong direction on those two graphs though.

**Comment by bsmithio** [comment]
WAN Bandwidth
![](https://i.imgur.com/iG6LVdx.png)

LAN Bandwidth with those settings
![](https://i.imgur.com/1foYcWR.png)

Is this the intention? 

I initially reversed them because this was my desired panel

LAN Bandwidth with reversed tx/rx
![](https://i.imgur.com/crgMP2N.png)

Very well could be that I'm getting reversed data for some reason.

**Comment by tonygauderman** [comment]
Yea, on LAN1, I would have expected IN to be from the LAN and OUT to be towards the LAN, the way the graphs read, it was showing the traffic being transmitted out towards the LAN as inbound.  If that’s not what you want to see, it’s easy to adjust, or get used to looking at it that way, it’s just not what I expected!

From: Brendan Smith ***@***.***>
Date: Tuesday, February 22, 2022 at 12:59 PM
To: bsmithio/OPNsense-Dashboard ***@***.***>
Cc: Tony Gauderman ***@***.***>, Author ***@***.***>
Subject: Re: [bsmithio/OPNsense-Dashboard] LAN vs. WAN data (Discussion #20)

WAN Bandwidth
[https://camo.githubusercontent.com/f1a2a40ab91d25b98860d4663f5e8e67b23d905c347af9ca98e0ae78bef1a4fd/68747470733a2f2f692e696d6775722e636f6d2f6947364c5664782e706e67]<https://camo.githubusercontent.com/f1a2a40ab91d25b98860d4663f5e8e67b23d905c347af9ca98e0ae78bef1a4fd/68747470733a2f2f692e696d6775722e636f6d2f6947364c5664782e706e67>

LAN Bandwidth with those settings
[https://camo.githubusercontent.com/fb5606ddd4c18faf458a7e0bd56e0655807c1cf3dd4373ecde36284f2bd7fafd/68747470733a2f2f692e696d6775722e636f6d2f31666f596357522e706e67]<https://camo.githubusercontent.com/fb5606ddd4c18faf458a7e0bd56e0655807c1cf3dd4373ecde36284f2bd7fafd/68747470733a2f2f692e696d6775722e636f6d2f31666f596357522e706e67>

Is this the intention?

I initially reversed them because this was my desired panel

LAN Bandwidth with reversed tx/rx
[https://camo.githubusercontent.com/4ae586233b3356b5926b6b9374596a253a2f5957263feb1fd866064c5fc43392/68747470733a2f2f692e696d6775722e636f6d2f6372674d50324e2e706e67]<https://camo.githubusercontent.com/4ae586233b3356b5926b6b9374596a253a2f5957263feb1fd866064c5fc43392/68747470733a2f2f692e696d6775722e636f6d2f6372674d50324e2e706e67>

—
Reply to this email directly, view it on GitHub<https://github.com/bsmithio/OPNsense-Dashboard/discussions/20#discussioncomment-2231098>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFTMJXYSQZ7HKV5TIELOIZTU4PMHHANCNFSM5PB5KXRQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
